analytics/pubstack/pubstack_module.go

```diff
@@ -200,6 +200,8 @@ func (p *PubstackModule) LogAmpObject(ao *analytics.AmpObject) {
 	p.eventChannels[amp].Push(payload)
 }
 
+func (p *PubstackModule) Shutdown() {}
```

Comment:
Should we `closeAllEventChannels()` or `close(p.stopCh)`?
```
202
203 - func (p *PubstackModule) Shutdown() {}
    + func (p *PubstackModule) Shutdown() {
    + }
    +    p.closeAllEventChannels()
    +    close(p.stopCh)
    + }
204  
205   func (p *PubstackModule) start(c <-chan *Configuration) {
analytics/pubstack/pubstack_module.go
```

Reply:
@guscarreon after inspection, i would say, leave it no-op, see the comments